sound/soc/sof/intel/cnl.c

```diff
@@ -422,6 +422,7 @@ int sof_cnl_ops_init(struct snd_sof_dev *sdev)
 
 		/* debug */
 		sof_cnl_ops.ipc_dump	= cnl_ipc4_dump;
+		sof_cnl_ops.dbg_dump	= hda_ipc4_dsp_dump;
```

Comment:
I don't think CNL will work either, we have IPC4 support in SOF only for TGL/MTL/LNL

Reply:
removed